subcell_pipeline/analysis/tomography_data/tomography_data.py

```diff
@@ -272,4 +329,6 @@ def plot_tomography_data_by_dataset(data: pd.DataFrame) -> None:
             ax[1].plot(fiber["xpos"], fiber["zpos"], marker="o", ms=1, lw=1)
             ax[2].plot(fiber["ypos"], fiber["zpos"], marker="o", ms=1, lw=1)
 
-        plt.show()
+    plt.savefig(local_save_path)
+    image: np.ndarray = imageio.imread(local_save_path)
+    save_image_to_s3(bucket, output_key, image)
```

Comment:
oh I tried to use save image but had to avoid it because I didn't want to depend on bioio for the OmeTiffWriter. It would help if this package was documented :)

Reply:
Haha yes I've been slowly working on the docs :)

The default install of `io-collection` does not include the `bioio` install, so should be okay.